test/jdk/sun/security/x509/Extensions/DefaultCriticality.java

```diff
@@ -25,22 +25,30 @@
  * @test
  * @summary Change default criticality of policy mappings and policy constraints
             certificate extensions
- * @bug 8059916
+ * @bug 8059916 8296742
```

Comment:
I don't think the bugid needs to be listed here. This has nothing to do with what is being tested, it is just a side-effect of the internal API behavior changing, and details can be seen in the commit log, if needed.